### PR TITLE
Bring python 3.6

### PR DESCRIPTION
**Pull request by ticosax**
Run tests against python3.6 to expose a bug with defusedxml

### CODE REVIEW

**Comment by ticosax** [comment]
Ready for review.

File: social_core/tests/backends/test_saml.py [hunk 1]
```diff
@@ -26,8 +29,8 @@
 DATA_DIR = path.join(path.dirname(__file__), 'data')
 
 
-@unittest2.skipIf('TRAVIS' in os.environ and sys.version_info[:2] == (3, 5),
-                  'Travis-ci segfaults in Python 3.5 probably due to a bad ' \
+@unittest2.skipIf('TRAVIS' in os.environ,
```
Comment (dotsbb) [inline social_core/tests/backends/test_saml.py]: Strange thing, [those tests](https://github.com/python-social-auth/social-core/pull/13/commits/55a03b65df64dd6a9e62f2af49b24e0c0b530022#diff-5bf0a2e8ccd1bff565f8367d724e1d11R72) passed in my PR https://travis-ci.org/python-social-auth/social-core/jobs/188914770

Comment (ticosax) [inline social_core/tests/backends/test_saml.py]: Thanks for the follow up. I will try to find the difference, and revert this commit eventually.

Comment (ticosax) [inline social_core/tests/backends/test_saml.py]: I gave up. It segfaults only when run within tox and on travis machine.
It seems #13 is better after all.

**Comment by omab** [comment]
@ticosax I've manually merged it, thanks for the work, but I decided to disable `py36` until python3-openid properly supports it, open-id is one of the main dependencies of this library, and while it's not properly supported on python3-openid, I can't consider it properly supported by this lib. Once that's corrected I'll re-enabled the support.

Thanks!

**Comment by ticosax** [comment]
thank you @omab, it makes sense.
For the record I raised this concern also on [python3-openid](https://github.com/necaris/python3-openid/pull/34)

**Pull request merged**